actioncable/test/remote_connection_test.rb

```diff
@@ -0,0 +1,54 @@
+require 'test_helper'
+require 'stubs/test_server'
+require 'minitest/autorun'
+
+class ActionCable::RemoteConnections::RemoteConnection::BaseTest < ActionCable::TestCase
```

Comment:
In fact, this whole class is private and we should only test it using `RemoteConnections`.